packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -400,6 +400,24 @@ export class LoadBalancer extends Resource implements IConnectable {
   }
 }
 
+/**
+ * An EC2 instance that is the target for load balancing
+ *
+ * If you register a target of this type, you are responsible for making
+ * sure the load balancer's security group can connect to the instance.
+ */
+export class InstanceTarget implements ILoadBalancerTarget {
+  /**
+   * Create a new Instance target.
+   *
+   * @param instance Instance to register to.
+   * @param connections The network connections associated with this resource.
+   */
+  constructor(public readonly instance: Instance, public readonly connections: Connections) {
+  }
+  public attachToClassicLB(_loadBalancer: LoadBalancer): void {
+  }
```

Comment:
This method isn't tested (and looks to be the important part). Also, there's no implementation? How are we expecting it to be attached?